Sources/SwiftGRPC/Core/Channel.swift

```diff
@@ -152,12 +152,13 @@ private extension Channel {
             }
             
             self.lastState = newState
-          case .queueShutdown:
-            break spinloop
           case .queueTimeout:
-            continue spinloop
+            continue
+          case .queueShutdown:
+            return
           default:
-            break spinloop
+            print("Event's completion type is `unknown`")
```

Comment:
Please also improve the formulation of this error as above.

Reply:
To be honest, I'm not fan of having `print` statements all over the place. Especially in libraries. I'd rather not have these in here. I'm going to remove them and feel free to put them back in another PR if you really want them.

Reply:
Fair enough. For what it's worth, I have a task in #189 to add a proper logging class. At that point, it should become much easier to properly detect and notice such errors (and possibly investigate why they happen).